stories/topics/ref-aws-additional-configs-replace-tls-certificate.adoc

```diff
@@ -0,0 +1,21 @@
+[id="ref-aws-additional-configs-replace-tls-certificate"]
+
+= Replacing {ControllerName} and {HubName} EC2 instances SSL/TLS certificate and key
+
+By default, EC2 instances are secured with a self-signed SSL/TLS certificate with a validity period of ten years.
+Once the certificate expires or you want EC2 instances to use your own certificate, you are required to replace the SSL/TLS certificate and key.
+
+.Procedure
+. Locate the correct secret for the CloudFormation Stack by opening the AWS Secrets Manager console and selecting the secret with the name _<stack_name>-pulp_cert_.
```

Comment:
Navigate to the AWS Secrets Manager console.
Locate the secret for the CloudFormation Stack with the name <stack_name>-pulp_cert (markup as before)

Reply:
<img width="1112" alt="image" src="https://github.com/ansible/aap-docs/assets/60555139/66345a9b-79a7-4b1d-80fd-dcf69e950795">